docs/source/usecases/s3globalgrants.rst

```diff
@@ -2,7 +2,7 @@ S3 - Global Grants
 ==================
 
 Scan buckets that allow for global access in their
-ACLs and report them and the ACL permissions granted.
+ACLs and delete the associated ACL permissions.
```

Comment:
per gitter convo, this doc wasn't clear on if the global-grant was being deleted